apps/designer/app/designer/shared/assets/types.ts

```diff
@@ -5,39 +5,28 @@ type PreviewAsset = Pick<
   "path" | "name" | "id" | "format" | "description"
 >;
 
-export type UploadedClientAsset = {
+export type UploadedAssetContainer = {
   status: "uploaded";
   asset: Asset;
-  preview: PreviewAsset | undefined;
 };
 
-export type UploadingClientAsset = {
+export type UploadingAssetContainer = {
   status: "uploading";
-  asset: Asset | undefined;
-  preview: PreviewAsset;
+  asset: Asset | PreviewAsset;
```

Comment:
btw since we call it container, maybe asset could be "value" but its up to you, not sure

Reply:
In some places it is used like `const { asset } = container`;  that looks better than `value`
In some `assetContainer.asset` that looks worse than `assetContainer.value`
Imo asset is better for search etc, as `value` is too wide a keyword